packages/react-tinacms-inline/src/styles/focus-ring.tsx

```diff
@@ -35,7 +35,10 @@ export interface StyledFocusRingProps
 
 export interface FocusRingProps {
   name?: string
-  children: React.ReactChild | React.ReactChild[]
+  children:
+    | React.ReactChild
```

Comment:
We could consider changing the children type to 
```
React.ReactNode | ((active: boolean) => React.ReactNode)
```
ReactNode allows multiple elements, strings, numbers, fragments, portals and probably a few more things. ReactChild only allows strings, numbers, and elements.